weather.py

```diff
@@ -57,7 +66,21 @@ def load_data_from_csv(csv_file):
     Returns:
         A list of lists, where each sublist is a (non-empty) line in the csv file.
     """
-    pass
+    with open (csv_file) as load_data:
+        data = csv.reader(load_data)
+        next(data)
+        data_list = []
+        data_b_c = []
+        for item in data:
+            if not (item):
+                continue
+            elif (item):
+                data_list.append(item)
+        for bc in data_list:
+            data_b_c.append([bc[0], int(bc[1]), int(bc[2])])
+        for item in data_b_c:
+            data_list.append(item)
```

Comment:
Could condense these `for` loops into a single loop, e.g.
```suggestion
        for item in data:
            if not (item):
                continue
            elif (item):
                data_list.append([bc[0], int(bc[1]), int(bc[2])])
```

Taking this a step further, we could make this a single `if` statement e.g.
```suggestion
        for item in data:
            if (item):
                data_list.append([bc[0], int(bc[1]), int(bc[2])])
```